cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `cinema.urls` and `user.urls` modules have the `app_name` attribute defined to support the use of namespaces in the `include` function.